testing/docker-master/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM ubuntu:xenial
+
+RUN apt update && apt dist-upgrade -y && apt install -y git && apt clean
+
+# enable multiverse as snapcraft cleanbuild does.
+RUN sed -i 's/ universe/ universe multiverse/' /etc/apt/sources.list
+
+# build and install snapcraft from master.
+git clone https://github.com/snapcore/snapcraft
+cd snapcraft
+apt build-dep -y ./
```

Comment:
I would probably clean the build-dep in the image after building the package to keep the docker container as close to a vanilla install as possible.